api-spec/STAC.yaml

```diff
@@ -928,14 +928,10 @@ components:
     itemCollectionLinks:
       type: array
       description: >-
-        An array of links. Can be used for pagination, e.g. by providing a link
-        with the `next` relation type.
+        An array of links to resources related to this collection.
       items:
         $ref: '#/components/schemas/link'
-      example:
-        - rel: next
-          href: >-
-            http://api.cool-sat.com/query/gasd312fsaeg/ANsXtp9mrqN0yrKWhf-y2PUpHRLQb1GT-mtxNcXou8TwkXhi1Jbk
+
```

Comment:
removed this, as it is now a confusing example with the `next` parameter and search:meta field.  We should replace it at some point with a good example.

Reply:
Can you come up with a good example? I'm always concerned that we forget to add it in again if we just remove it without creating an issue or so...

Reply:
I think something like `http://api.cool-sat.com/stac/search?next=ANsXtp9mrqN0yrKWhf-y2PUpHRLQb1GT-mtxNcXou8TwkXhi1Jbk` should work.

Reply:
added.